ext/opentelemetry-exporter-cloud-monitoring/tests/test_cloud_monitoring.py

```diff
@@ -289,3 +289,43 @@ def test_export(self):
                 mock.call(self.project_name, [series3]),
             ]
         )
+
+    def test_unique_identifier(self):
+        client = mock.Mock()
+        exporter1 = CloudMonitoringMetricsExporter(
+            project_id=self.project_id,
+            client=client,
+            add_unique_identifier=True,
+        )
+        exporter2 = CloudMonitoringMetricsExporter(
+            project_id=self.project_id,
+            client=client,
+            add_unique_identifier=True,
+        )
+        exporter1.project_name = self.project_name
+        exporter2.project_name = self.project_name
+
+        client.create_metric_descriptor.return_value = MetricDescriptor(
+            **{
+                "name": None,
+                "type": "custom.googleapis.com/OpenTelemetry/name",
+                "display_name": "name",
+                "description": "description",
+                "labels": [],
+                "metric_kind": "GAUGE",
+                "value_type": "DOUBLE",
+            }
+        )
+
+        sum_agg_one = SumAggregator()
+        sum_agg_one.checkpoint = 1
+        sum_agg_one.last_update_timestamp = (WRITE_INTERVAL + 1) * 1e9
```

Comment:
Is this line needed? I don't quite see how this ties into the test, in particular why the `WRITE_INTERVAL` is involved.

Reply:
Whoops, remnant from before